include/hip/hcc_detail/hiprtc.h

```diff
@@ -41,29 +45,22 @@ enum hiprtcResult {
 
 const char* hiprtcGetErrorString(hiprtcResult result);
 
-inline
-hiprtcResult hiprtcVersion(int* major, int* minor) noexcept
-{   // TODO: NVRTC versioning is somewhat unclear.
-    if (!major || !minor) return HIPRTC_ERROR_INVALID_INPUT;
-
-    // TODO: this should be generic / set by the build infrastructure.
-    *major = 9;
-    *minor = 0;
 
-    return HIPRTC_SUCCESS;
-}
```

Comment:
We are missing the implementation for hiprtcVersion() in src/hiprtc.cpp. This implementation needs to move there.

Reply:
Done